src/app/components/NavBar.css

```diff
@@ -4,15 +4,15 @@ a {
     color: white;
 }
 
-a:visited {
+/* a:visited {
     text-decoration: none;
     color: white;
 }
 
 a:hover {
     text-decoration: underline;
     color: white;
-}
+} */
```

Comment:
I will say that I think commenting things out like this (and [this](https://github.com/DevDogs-UGA/DevDogs-Website/pull/42/files#diff-976ddd751899c9f095b17db0bbdc8da3eec0728d8a25d6d1ee6ed45dbc7b4f6dR58-R60)) is bad practice - Git is keeping track of all your changes, so things like this aren't lost forever if you just delete them. Commenting them out just clutters the reading experience.